src/components/User/Sales/SaleRequests/components/SaleRequestManager/SaleRequestManager.vue

```diff
@@ -123,7 +124,14 @@ export default {
     async assignRequest (request) {
       try {
         this.request.sale = request
-        this.request.token = await this.getTokenRequest(this.request.sale.baseAsset)
+        this.request.token = await Sdk.horizon.request.getAllForAssets({
+          asset: this.request.sale.baseAsset.code,
+          requestor: this.request.sale.baseAsset.requestor,
+          state: this.request.sale.baseAsset.state,
+          reviewer: this.request.sale.baseAsset.reviewer,
+          order: 'desc'
+        }).data[0]
+        console.log(this.request.token)
```

Comment:
remove log

Reply:
also here is a bug: you're not awaiting the promise, you're trying to await it's `data[0]` property